rfc/0002-gotrue-unfork.md

```diff
@@ -0,0 +1,50 @@
+- Feature Name: `gotrue_unfork`
+- Start Date: 2021-10-07
+- RFC PR: [supabase/rfcs#0002](https://github.com/supabase/rfcs/pull/0002)
+- Supabase Issue: [supabase/supabase#0000](https://github.com/supabase/supabase/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Our [Auth server](https://github.com/supabase/gotrue) is currently a fork of [GoTrue by Netlify](https://github.com/netlify/gotrue). This RFC explores the possibility of "unforking".
+
+# Motivation
+[motivation]: #motivation
+
+There are several drawbacks to the current fork:
+
+- *Search*: Due to the way GitHub operates, if you search in our repo it will instead search in the fork. This makes it difficult to find issues/code/PRs.
```

Comment:
It looks like GitHub have solved this now? If I search in our repo it no longer redirects to Netlify's

Reply:
It still doesn't do code search for me (e.g. `discord` which should be in our README but not Netlify's).

There's [this note](https://docs.github.com/en/github-ae@latest/search-github/searching-on-github/searching-in-forks) in GitHub docs:

> Forks are only indexed for code search when they have more stars than the parent repository. You will not be able to search the code in a fork that has less stars than its parent.